src/model.py

```diff
@@ -0,0 +1,73 @@
+import tensorflow as tf
+from src.config_constants import DENSE_UNITS
+from src.constants import IMG_LEN, NUM_CHANNELS
+
+
+class BaldOrNotModel(tf.keras.Model):
+    """
+    Binary classification model to determine if a person in an image is bald
```

Comment:
should be here or under init?

Reply:
As I see here https://www.programiz.com/python-programming/docstrings docstring should be under class. There could be another one under init but in our case it would be nearly identical

Reply:
OK, cool